queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '01-01-2016' AND '01-12-2019';
```

Comment:
- [ ] I think to include all of 2019, the date_of_birth should be between `01-01-2016 AND 31-12-2019`, please fix the range to match with the requirements(`List the name of all animals born between 2016 and 2019.`).

Reply:
I didn't notice, thanks!